dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,8 +407,27 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+
+    // check if application.conf is present in service
+    def isValidProject(p: ProjectRef) = {
+
+      val prj = Project.getProject(p, buildStructure.value)
+
+      val r: PathFinder = {
+        (new File(prj.get.base.getPath) / "src") ** "application.conf"
```

Comment:
I believe not :( 
I tested yesterday evening. I hoped too that it works. that's why I asked the question this morning. 
I going to search another way to reach the resourcesDirectories setting, 
If you have an idea, you're welcome :D :)